pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitDaoImpl.java

```diff
@@ -66,19 +68,31 @@ public class KitDaoImpl implements KitDao {
             + "(SELECT 1 FROM "
             + "ddp_kit_tracking tracking "
             + "WHERE tracking.kit_label = ?) AS existing_rows";
-    private static final String UPDATE_KIT_REQUEST = "UPDATE ddp_kit SET "
+
+    private static final String UPDATE_KIT_REQUEST_DDP_LABEL = "update ddp_kit_request set ddp_label = ? where "
```

Comment:
Now that the method is removed do we need this one?

Reply:
Indeed we don't